packages/SwingSet/docs/virtual-objects.md

```diff
@@ -96,22 +93,22 @@ Additional important details:
 
 - A virtual object's state may include references to other virtual objects. The latter objects will be persisted separately and only deserialized as needed, so "swapping in" a virtual object that references other virtual objects does not entail swapping in the entire associated object graph.
 
-- The `self` object returned by the `instanceKitMaker` function _is_ the in-memory representation of the virtual object, so you can capture it inside any of the body methods (as well as the `init` function) in order to pass the virtual object to somebody else.  So you can write things like the following:
+- The object returned by the `actualize` function _is_ the in-memory representation of the virtual object, so you can capture it inside any of the body methods in order to pass the virtual object to somebody else.  So you can write things like the following:
 
 ```javascript
-  function thingKitMaker(state) {
-    const self = Far('thing', {
-      sendMeTo(somebodyElse) {
-        somebodyElse.hereIAm(self);
-      },
-      ... // and so on
-    });
-    function init(someParam) {
-      otherObject.doSomethingWithMe(self, someParam);
-      state.foo = 47;
+  const makeThing = defineKind(
+    'thing',
+    () => ({
+      someProperty: 47,
+    }),
+    state => {
+      const self = {
+        sendMeTo(somebodyElse) {
+          somebodyElse.hereIAm(state.someProperty, self);
```

Comment:
I *think* this runs into the same problem we previously had, if someone decides to make that cross-linkage `hereIAm` call at actualization time rather than waiting for a method to be invoked, and `self` is not yet registered anywhere. OTOH, if we're doing the `Far` marker inside the VOM, then `self` at this point is not passable (it isn't even hardened yet, nor marked as Far), which might catch this mistake early. So that's a clear improvement.